types/react-star-ratings/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "esModuleInterop": true,
```

Comment:
Please remove this to show that this package will work with old code bases that haven't turned it on yet.